src/print.jl

```diff
@@ -352,10 +352,10 @@ function pretty_table(io::IO, table; kwargs...)
     # Get the data.
     #
     # If `table` is not compatible with Tables.jl, then an error will be thrown.
-    data = Tables.columns(table)
-
+    Tables.columns(table)  
```

Comment:
Any particular reason why we need this?

Reply:
I mean, why call `Tables.columns` if we are converting to matrix below.

Reply:
we don't need it I removed it. this PR is ready to go